Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1898,13 +1908,22 @@ void CGContextSetPatternPhase(CGContextRef context, CGSize phase) {
 #pragma endregion
 
 #pragma region Drawing Operations - Text
+
+// Attributes are needed in DrawGlyphRuns but names are defined in CoreText
+const CFStringRef _kCGForegroundColorAttributeName = static_cast<CFStringRef>(@"NSForegroundColor");
```

Comment:
I think we prefer CFSTR() when we're in the Core_____ frameworks

---
In reply to: [100870707](https://github.com/Microsoft/WinObjC/pull/1933#discussion_r100870707) [](ancestors = 100870707)